common/types/src/main/java/io/helidon/common/types/TypeValues.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.common.types;
+
+/**
+ * Constants to be used with {@link io.helidon.common.types.TypeInfo} and {@link io.helidon.common.types.TypedElementInfo}.
+ */
+public final class TypeValues {
+    /**
+     * The {@code public} modifier.
+     */
+    public static final String MODIFIER_PUBLIC = "public";
```

Comment:
seems we are repeating these in several places. can't we consolidate?

Reply:
We should be using this constant everywhere. we may need to run a check on our processors (as there was a time when modifier constdants were not working correctly, some uses may be explicit)